collaborations/dark-tides-of-bilgewater/rogue-wildcard.xml

```diff
@@ -0,0 +1,152 @@
+		</rules>
+	</element>
+
+	<element name="Tricks Up the Sleeve" type="Archetype Feature" source="Legends of Runeterra: Dark Tides of Bilgewater" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_TRICKS_UP_THE_SLEEVE">
+		<description>
+			<p>Your penchant for games has afforded you the ability to subtly manipulate fortune to your favor. When you choose this archetype at 3rd level, you learn the guidance cantrip. Starting at 9th level, it has a range of 30 feet for you, and you can cast it as a bonus action.</p>
+		</description>
+		<sheet>
+			<description>You learn the guidance cantrip. Starting at 9th level, it has a range of 30 feet for you, and you can cast it as a bonus action.</description>
+		</sheet>
+		<rules>
+			<grant type="Spell" id="ID_PHB_SPELL_GUIDANCE" />
+		</rules>
+	</element>
+
+	<element name="Wild Card's Gambit" type="Archetype Feature" source="Legends of Runeterra: Dark Tides of Bilgewater" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_WILD_CARDS_GAMBIT">
+		<description>
+			<p>Also at 3rd level, you gain proficiency with one of the following gaming sets: dice set, dragonchess set, or playing card set.</p>
+			<p class="indent">The gaming set you choose grants you a Wild Card’s Gambit, as detailed below. If you are proficient in multiple types of gaming sets, you must choose which gambit to use when you gain this feature. You can change your choice of gambit whenever you gain a level in this class.</p>
+		</description>
+		<rules>
+			<select type="Archetype Feature" name="Wild Cards Gambit" supports="Wild Cards Gambit, Wild Card Rogue" number="1" />
+		</rules>
+	</element>
+
+	<element name="Shifting the Odds" type="Archetype Feature" source="Legends of Runeterra: Dark Tides of Bilgewater" id="ID_DDB_LRDTOB_ARCHETYPE_FEATURE_WILD_CARD_SHIFTING_THE_ODDS">
+		<description>
+			<p>Starting at 9th level, you are acutely aware of how to quit when you’re ahead, vanishing in a flash when the odds begin to turn against you. As a bonus action, you can disappear with a dramatic flourish. Each creature within 10 feet of you must make a Dexterity saving throw (DC 8 + your proficiency bonus + your Charisma modifier), taking 4d10 force damage on a failed save or half as much on a successful one.</p>
+			<p class="indent">You then teleport yourself to an unoccupied space that you can see within 120 feet of your original location. Once you have used this ability, you cannot use it again until you have finished a short or long rest.</p>
+		</description>
+		<sheet action="Bonus Action" usage="1/Short Rest">
+			<description>As a bonus action, you can disappear with a dramatic flourish. Each creature within 10 feet of you must make a Dexterity saving throw (DC 8 + your proficiency bonus + your Charisma modifier), taking 4d10 force damage on a failed save or half as much on a successful one.You then teleport yourself to an unoccupied space that you can see within 120 feet of your original location.</description>

```

Comment:
"successful one.You" need a space in between. You also need to make a stat for: "DC 8 + your proficiency bonus + your Charisma modifier."